zanata-model/src/main/java/org/zanata/model/HAccountRole.java

```diff
@@ -74,6 +74,7 @@ public String getName() {
         return groups;
     }
 
+    // TODO [CDI] check whether we actually use this field (doesn't seem to in prod db and in code)
     @RoleConditional
     public boolean isConditional() {
```

Comment:
@carlosmunoz Have you got any idea what `RoleConditional` does?  It seems to predate the role stuff you implemented for Fedora authentication.